packages/react/tests/e2e-storybook/cypress/integration/CardSectionSimple/CardSectionSimple.e2e.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright IBM Corp. 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Sets the correct path
+ *
+ * @param default - Path to default variant
+ * @param withCta - path to variant with CTA card
+ * @private
+ */
+const _paths = {
+  default: 'iframe.html?id=components-card-section-simple--default',
+  withCta: 'iframe.html?id=components-card-section-simple--with-cta',
+};
+
+/**
+ * Collection of all tests for dds-card-section-simple
+ *
+ * @function checkForBlocklink - Asserts a full-sized pseudoelement from the footer link
+ * @function checkForTitlePosition - Asserts left-aligned title (16 || 32)px from left edge
+ * @function checkForCardContent - Asserts that all card child elements are rendered in shadowRoot slots
+ * @private
+ */
+const _tests = {
+  checkForBlocklink: () => {
+    cy.get('.bx--link.bx--card__footer').each($els => {
+      const win = $els[0].ownerDocument.defaultView;
+      const after = win.getComputedStyle($els[0], ':after');
+      const positionValue = after.getPropertyValue('position');
+      const insetValue = after.getPropertyValue('inset');
+
+      expect(positionValue).to.eq('absolute');
+      expect(insetValue).to.eq('0px');
+    });
+  },
+  checkForTitlePosition: () => {
+    cy.get('.bx--content-section__heading').then(heading => {
+      expect(
+        heading.offset().left == 16 || heading.offset().left == 32
+      ).to.be.eq(true);
+      expect(heading.css('textAlign')).to.be.eq('start');
+    });
+  },
+  checkForCardContent: () => {
+    cy.get('.bx--card-group__card').each($card => {
+      expect(Cypress.dom.isVisible($card.find('.bx--card__heading'))).to.be.eq(
+        true
+      );
+      expect(Cypress.dom.isVisible($card.find('.bx--card__copy'))).to.be.eq(
+        true
+      );
+      expect(Cypress.dom.isVisible($card.find('.bx--card__footer'))).to.be.eq(
+        true
+      );
+    });
+  },
+  checkThemes: () => {
+    cy.carbonThemesScreenshot();
+  },
+};
+
+describe('dds-card-section-simple | default (desktop)', () => {
```

Comment:
Hi @andy-blum! Tests look good just one small thing for react changing the test names to: 
```suggestion
describe('DDSCardSectionSimple | default (desktop)', () => {
```